setup.py

```diff
@@ -410,7 +410,8 @@ def run(self):
             "google_drive_task = dispatch.plugins.dispatch_google.drive.plugin:GoogleDriveTaskPlugin",
             "google_gmail_conversation = dispatch.plugins.dispatch_google.gmail.plugin:GoogleGmailConversationPlugin",
             "google_groups_participants = dispatch.plugins.dispatch_google.groups.plugin:GoogleGroupParticipantGroupPlugin",
-            "jira_ticket = dispatch.plugins.dispatch_jira.plugin:JiraTicketPlugin",
+            #"jira_ticket = dispatch.plugins.dispatch_jira.plugin:JiraTicketPlugin",
```

Comment:
No need to disable the Jira plugin. You can tell Dispatch to use your Gitlab plugin by adding `INCIDENT_PLUGIN_TICKET_SLUG="gitlab-ticket"` to your `.env` file. Dispatch uses the Jira one by default if none are specified. See https://github.com/Netflix/dispatch/blob/develop/src/dispatch/config.py#L90

Reply:
Fixed, thx for the infos.